pkg/lib/cockpit-po-plugin.js

```diff
@@ -5,7 +5,7 @@ const gettext_parser = require('gettext-parser');
 const Jed = require('jed');
 const webpack = require('webpack');
 
-const srcdir = process.env.SRCDIR || path.resolve(__dirname, '..', '..');
+const srcdir = process.env.SRCDIR || '.';
```

Comment:
Here you need ../../

Reply:
Hmm, why? `__dirname` used to be 'pkg/lib', and the two .. just undid that.

Reply:
It's supposed to be the top-level source directory. So it could actually be that this breaks for out-of-tree builds, but image-prepare does that, and it seems happy enough.